tests/presses/test_presses.py

```diff
@@ -29,9 +30,18 @@ def test_think_inner_press(unit_test_model):  # noqa: F811
 
 
 def test_presses_run(unit_test_model):  # noqa: F811
-    for cls in [KnormPress, ExpectedAttentionPress, RandomPress, StreamingLLMPress, SnapKVPress, TOVAPress, ThinKPress]:
+    for cls in [
+        KnormPress,
+        ExpectedAttentionPress,
+        RandomPress,
+        SimLayerKVPress,
+        StreamingLLMPress,
+        SnapKVPress,
+        TOVAPress,
+        ThinKPress,
+    ]:
         for compression_ratio in [0.2, 0.4, 0.6, 0.8]:
-            press = cls(compression_ratio=compression_ratio)
+            press = cls(compression_ratio)
```

Comment:
I'd leave `compression_ratio=compression_ratio` in.

Reply:
I made it more explicit: 

```python
        for value in [0.2, 0.4, 0.6, 0.8]:
            if cls == SimLayerKVPress:
                press = cls(lazy_threshold=value)
            else:
                press = cls(compression_ratio=value)
```